### DISCUSSION TITLE
What should the threshold salt file be called under /opt/so/saltstack/local/pillar/thresholding/?

### DISCUSSION THREAD

**Discussion by w4rc0n** [Security-Onion-Solutions/securityonion#2166] [General]
I'm looking at the docs at: https://docs.securityonion.net/en/2.3/managing-alerts.html

I don't have any files under  `/opt/so/saltstack/local/pillar/thresholding/` on my manager node. The docs reference 2 files that contain examples, but I'm not sure what the actual file should be called should someone want to set up threshold rules.

**Comment by cm-ops** [comment]
The pillar example is in `/opt/so/saltstack/default/pillar/thresholding`. The example in the docs is the same. You use that salt pillar and add it to one of the .sls files below.

If you want to set up threshold rules to apply the threshold to all nodes, place the pillar in `/opt/so/saltstack/local/pillar/global.sls`. If you want to apply the threshold to a single node, place the pillar in /`opt/so/saltstack/local/pillar/minions/<MINION_ID>.sls`. Then restart suricata with `sudo salt 'sensorname' cmd.run 'so-suricata-restart'`.

**Reply by w4rc0n** [reply]
Thank you
